day_two_guide/topics/proc_scaling-etcd-ansible.adoc

```diff
@@ -64,8 +64,18 @@ master-2.example.com
 etcd0.example.com
 ----
 
+. If you use the service catalog, you must update its list of etcd servers:
++
+[source,bash]
+----
+$ oc edit ds apiserver -n kube-service-catalog
+----
++
+Add the etcd URL for the new etcd node to the `--etcd-servers` argument. This
```

Comment:
Do you need "etcd" in front of URL here? Could it just say:

`Add the URL for the new etcd node to...`